index.bs

```diff
@@ -1341,6 +1343,31 @@ To <dfn for="XRView">obtain the projection matrix</dfn> for a given {{XRView}} |
 </div>
 
 
+Additional Views {#additional-views}
+----------
+
+While most devices will either have a single "mono" [=view=] or two "stereo" [=views=], it is possible for devices to have <dfn>additional views</dfn> which do not fall into these categories.
+
+<div class="example">For example, there can be a [=first-person observer view=] used for video capture, or "quad views" where there are two views per eye with differing resolution and fields of view. There can also be CAVE systems which have many, many views surrounding the user.</div>
+
+While content should be written to assume that there may be any number of views, we expect a significant amount of content to make incorrect assumptions about the {{XRViewerPose/views}} array and thus break when presented with more than two views.
+
+Because user-agents may have the ability to use mechanisms like reprojection to render to these [=additional views=] in lieu of the content, it is desirable to be able to distinguish between content that plans on handling these [=additional views=] itself and content that is either oblivious to the existence of such [=additional views=] or does not wish to deal with them.
+
+To provide for this, user-agents MAY support an <dfn>additional-views</dfn>" [=feature descriptor=] as a hint. Content enabling this feature is expected to:
```

Comment:
bikeshed: i don't have strong opinions on the name

Reply:
"secondary-views" is similar to the language OpenXR uses, however technically Varjo quad views are not secondary views since you always should render to them.